### ISSUE TITLE
Regression issue with combodate

### ISSUE THREAD

**Issue by josemonsalve2** [vitalets/angular-xeditable#706]
When trying to use combodate with version 0.8.1 there is an error with the new addition of .change. 

Error:
``` 
TypeError: combodate.$widget.find(...).bind(...).change is not a function
```

The problem is in the .change() addition to xeditable.js after line 432 (or nearby)

**Comment by josemonsalve2** [comment]
JsFiddle here

https://jsfiddle.net/josemonsalve2/u03z4kw2/

**Comment by ckosloski** [comment]
I don't see that error at the JSFiddle link above.  I tested in the most recent FF and all the editable links did not throw that error.
Is there a specific browser this error is occurring in?

**Comment by josemonsalve2** [comment]
Sorry I forgot to stage my changes in the jsFiddle as base. Now it should be updated with the issue

**Comment by ckosloski** [comment]
I think your fix from #705 solves the problem.

**Comment by ckosloski** [comment]
@eugef this issue can be closed, it was fixed by #709
